README.md

```diff
@@ -42,60 +42,69 @@ c.GitPlus.github_token = '<your-github-access-token>'
 
 After installation, start JupyterLab normally & you should see "Git-Plus" as a new menu item at the top.
 
+#### 2.1 Configuring a custom reviewnb URL
+
+Suppose your company uses a custom reviewnb URL, e.g. `https://reviewnb.yourdomain.com`. You can configure
+gitplus to link users to PRs/commits at this URL instead of the default one as follows:
+
+```python
+c.GitPlus.reviewnb_endpoint = "https://reviewnb.yourdomain.com/"
```

Comment:
Let's please call this `c.GitPlus.self_hosted_reviewnb_endpoint` to be explicit.

Reply:
Done